components/browser/engine-gecko-nightly/src/main/java/mozilla/components/browser/engine/gecko/GeckoEngine.kt

```diff
@@ -137,7 +138,7 @@ class GeckoEngine(
      * Creates a new Gecko-based EngineView.
      */
     override fun createView(context: Context, attrs: AttributeSet?): EngineView {
-        return GeckoEngineView(context, attrs)
+        return GeckoEngineView(context, attrs, actionSorter = actionSorter)
```

Comment:
The approach overall looks good to me. However it is a bit annoying that we will have to pass it to the `Engine` and here down to the `View` as well. Especially since that breaks the view constructor contract and so that doesn't work if you add the view to your XML.
Are there other options how we can get this passed down? Ideally we can set our sorter function on our custom delegate (or return it from there), is that something that could work?